src/frontend/src/components/common/Collapsible/index.tsx

```diff
@@ -11,8 +11,8 @@ const Collapsible = ({
   alignment,
   children,
   isVisible = false,
-  onOpenHandler,
```

Comment:
should keep these two props, it's my last pr added

Reply:
I thought we didn't need these anymore because we are trying to make these components more 'dumb.'